edgelet/iotedge/src/check/checks/container_connect_upstream.rs

```diff
@@ -166,14 +166,8 @@ impl ContainerConnectUpstream {
         }
 
         if &port == "443" {
-            let proxy = settings
-                .agent()
-                .env()
-                .get("https_proxy")
-                .map(std::string::String::as_str);
-            self.proxy = proxy.map(ToOwned::to_owned);
```

Comment:
In your changes, `self.proxy` is no longer being assigned. However, it looks as though `self.proxy`, though assigned, wasn't being used for anything anyway. Maybe it should be removed from `struct ContainerConnectUpstream`?

Reply:
>wasn't being used for anything anyway.

It was there to be serialized into the JSON output.

Reply:
Ah, I see. So @scb01 you need to bring back that assignment.

Reply:
Makes sense. Addressed.

Reply:
@scb01 Did you confirm that the JSON output from the check now contains the proxy address?

Reply:
Yep. Ran `iotedge check --output json` locally and verified that the output contains the configured value for proxy-uri ( or null if proxy-uri is not configured)